src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -2779,7 +1832,7 @@ bool processOnChangeConditionForSubscription
         getNotifier()->sendNotifyContextRequest(&ncr, notifyUrl, tenant, xauthToken, format);
         allCerV.release();
         ncr.contextElementResponseVector.release();
-        res.release();
+        resP->release();
```

Comment:
Same comment here (last comment about this... but if you search/grep for `resP->release()` you will catch them all).